translate/src/modules/translationform/utils/editFieldModes.ts

```diff
@@ -88,12 +88,20 @@ const printf =
 
 const pythonFormat = /^{[\w.[\]]*(![rsa])?(:.*?)?}/;
 
+// https://www.i18next.com/translation-function/interpolation
+const i18nextFormat =
+  /{{\s*[\w.[\]]*\s*(?:,\s*\w+(?:\([^)]*\))?)*\s*(?:![rsa])?(?::.*?)?\s*}}/;
```

Comment:
This does seem a bit complex. Are there cases where something simpler like this would not work, or would produce misleading results?
```suggestion
const i18nextFormat = /{{.*?}}/;
```

Reply:
I don't see why it should produce wrong ones.
Thanks for the suggestion.
Applied ✅ 

<sub>I guess using GPT made me silly and lazy that I didn't think of the most obvious one</sub>